### PR TITLE
[12.0][FIX] hr_attendance_autoclose: compute needs api.depends

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Fast-track.

### CODE REVIEW

File: hr_attendance_autoclose/models/hr_attendance.py [hunk 1]
```diff
@@ -10,6 +10,7 @@ class HrAttendance(models.Model):
     _inherit = "hr.attendance"
 
     @api.multi
+    @api.depends('check_out')
```
Comment (alexey-pelykh) [inline hr_attendance_autoclose/models/hr_attendance.py]: and `check_in`?

Comment (MiquelRForgeFlow) [reply hr_attendance_autoclose/models/hr_attendance.py]: No, it doesn't depend of `check_in`.

Comment (alexey-pelykh) [reply hr_attendance_autoclose/models/hr_attendance.py]: `attendance.check_out - attendance.check_in`?

Comment (alexey-pelykh) [reply hr_attendance_autoclose/models/hr_attendance.py]: @mreficent ^

**Comment by AaronHForgeFlow** [comment]
Does it need api.depends even if the field is not stored?

**Comment by jarroyomorales** [comment]
If you make it depend on `check_out` and the user doesn't check out, `open_worked_hours` will never be updated and the cron wont close it.

**Comment by AaronHForgeFlow** [comment]
I mean, if it is not stored, it is recomputed any time a user reads the information, isn't it?

File: hr_attendance_autoclose/models/hr_attendance.py (hunk 1 above)
Comment (etobella) [reply hr_attendance_autoclose/models/hr_attendance.py]: I think `check_in` is necessary

**Comment by alexey-pelykh** [comment]
let's follow https://www.odoo.com/documentation/12.0/reference/orm.html#computed-fields

**Comment by alexey-pelykh** [comment]
We should list literally all used fields in `dependson`

**Comment by AaronHForgeFlow** [comment]
Tested on runbot. Updating `check_in` field triggered the computation of the field. In any case, if we add the `api.depends` we should add also `check_in` in the `api.depends`.

File: hr_attendance_autoclose/models/hr_attendance.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply hr_attendance_autoclose/models/hr_attendance.py]: all of you are right, the check_in is necessary.

**Comment by MiquelRForgeFlow** [comment]
hey people, `check_in` added. please update review and merge :)

**Comment by alexey-pelykh** [comment]
but have you pushed? :)

**Review by alexey-pelykh — APPROVED** [review]

**Comment by alexey-pelykh** [comment]
fast-tracking as trivial

**Comment by MiquelRForgeFlow** [comment]
> but have you pushed? :)

there was some lag haha

**Review by etobella — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:

**Pull request merged**